apps/backend/src/middleware/enhanced-auth.middleware.ts

```
@@ -0,0 +1,321 @@
+      }
+      next();
+    });
+  }
+
+  // Return combined middleware
+  return (req: Request, res: Response, next: NextFunction) => {
+    let index = 0;
+
+    function runNext(): void {
+      if (index >= middlewares.length) {
+        return next();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)